data/brands/shop/pawnbroker.json

```diff
@@ -223,6 +223,7 @@
       "locationSet": {"include": ["ph"]},
       "tags": {
         "brand": "RD Pawnshop",
+        "brand:wikidata": "Q119851235",
```

Comment:
Looks like you created a duplicating wikidata QID for https://www.wikidata.org/wiki/Q118556221
Can you merge the new copy into the old one and update the PR?

Reply:
Thank you for catching the error. I'll fix that, and resubmit the changes.